app/build.gradle

```diff
@@ -86,8 +82,11 @@ dependencies {
     ksp(libs.hilt.compiler)
     ksp(libs.hilt.ext.compiler)
 
+    // Testing Dependencies
     androidTestImplementation libs.androidx.core.testing
     androidTestImplementation libs.bundles.androidx.test.junit
+    testImplementation libs.junit
+    androidTestImplementation libs.androidx.junit
```

Comment:
Why import separately when the bundled import at line 87 has already taken care of everything?